### PR TITLE
Dropout 2D function and Test in Paddle

### LINKED ISSUE #17475: dropout2d

### PR DESCRIPTION
**Pull request by HaiderSultanArc**
Close #17475

### CODE REVIEW

**Comment by DragosStoican** [comment]
Hi @HaiderSultanArc please also make sure that the formatting test passes or I cannot merge the PR. You can check the logs here on github or locally to see what went wrong!

**Comment by HaiderSultanArc** [comment]
@DragosStoican I fixed the formatting. In pre-commit, it showed that everything passed. But now it says "black" failed.
image

**Comment by DragosStoican** [comment]
When testing the function, we should ideally test for all valid dtypes, not only a subset. Can you use the key "valid" when generating dtypes and making sure the test is still passing? I added a comment on that line to help you understand what I'm referring to :)

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_paddle_common.py [hunk 1]
```diff
@@ -41,3 +41,50 @@ def test_paddle_cosine_similarity(
         x2=x[1],
         axis=axis,
     )
+
+
+# Dropout2d
+@handle_frontend_test(
+    fn_tree="paddle.nn.functional.common.dropout2d",
+    d_type_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```
[the remark this answers is not in the document]
Comment (HaiderSultanArc) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_paddle_common.py]: Got it

**Comment by DragosStoican** [comment]
Thank you for your contribution!
